AspNetCoreApplication/Controller/CategoriesController.cs

```diff
@@ -18,9 +18,17 @@ public class CategoriesController : ControllerBase
     {
         private readonly IRepository<Category> categoryRepository;
 
-        public CategoriesController(IRepository<Category> categoryRepository)
+        private readonly IBookCategoryRepository bookCategoryRepository;
```

Comment:
Field này hoàn toàn không được sử dụng trong class này

Reply:
done